rdr_service/model/bq_genomics.py

```diff
@@ -278,6 +278,7 @@ class BQGenomicManifestFileSchema(BQSchema):
     rdr_processing_complete_date = BQField('rdr_processing_complete_date', BQFieldTypeEnum.DATETIME,
                                            BQFieldModeEnum.NULLABLE)
     ignore_flag = BQField('ignore_flag', BQFieldTypeEnum.INTEGER, BQFieldModeEnum.REQUIRED)
+    file_name = BQField('file_name', BQFieldTypeEnum.STRING, BQFieldModeEnum.NULLABLE)
```

Comment:
Can you also add the field to this file:
rdr_service/resource/schemas/genomics.py

Reply:
👍